test/index.js

```diff
@@ -50,3 +50,4 @@ require('./typedef');
 require('./const');
 require('./default');
 require('./enum');
+require('./unrecognized-exception');
```

Comment:
Disregard. There is a ./errors.js where we keep the constructors for all the error types, which does typically use that error package. However, urecognized exception fits more squarely in the thrift exception category. Having its own module is fine.